recipes/optimum/assets/README.md

```diff
@@ -0,0 +1,33 @@
+# Special Notes
```

Comment:
This is very descriptive and you're explaining steps that should be normal in the conda-forge context. Please only include the patch and the meta.yaml.

These extra ones can trip up the bot and also grow the package.

Reply:
I will remove the rest of the files.